.ci/jenkins/Jenkinsfile.publish

```diff
@@ -0,0 +1,97 @@
+@Library('jenkins-pipeline-shared-libraries')_
+import org.kie.jenkins.MavenCommand
+
+AGENT_LABEL="kie-rhel7 && kie-mem4"
+MVN_TOOL="kie-maven-3.8.1"
+JDK_TOOL="kie-jdk11"
+REPO="kiegroup-website"
+ORGANIZATION="kiegroup"
+BASE_BRANCH="main"
+MAIL_RECIPIENT="jporter@redhat.com"
```

Comment:
is it fine to expose a personal email from here? :thinking: shouldn't it be moved to a global env variable?